src/components/User/Operations/OperationDetails.vue

```diff
@@ -12,12 +14,63 @@
 <script>
 import DetailsReader from '../../common/details/DetailsReader'
 
+// ToDo: get operation by operationId from server
+import moment from 'moment'
+import get from 'lodash/get'
+import { ErrorHandler } from '@/utils/ErrorHandler'
+import { ApiCallerFactory } from '@/api-caller-factory'
+import { clearObject } from '@/utils/clearObject'
+
 export default {
   components: {
     DetailsReader,
   },
   props: {
-    operation: { type: Object, required: true },
+    operationId: { type: String, required: true },
+  },
+  data: _ => ({
+    userId: null,
+    operation: {},
+  }),
+  async created () {
+    window.scroll(0, 0)
```

Comment:
Useless? This component used in the route, so when you go to the new route, the page automatically scrolls to the top. If I'm wrong, tell me about it pls and do not fix

Reply:
you are right, the scroll isn't needed